std/testing/asserts.ts

```diff
@@ -243,6 +243,64 @@ export function assertStrictEq(
   throw new AssertionError(message);
 }
 
+/**
+ * Make an assertion that `actual` greater than `expected` value. If not
+ * then thrown.
+ */
+export function assertToBeGreaterThan(
+  actual: number,
+  expected: number,
+  msg?: string
+): void {
```

Comment:
Is there a prior art for these APIs? 

It's three times as many chars to type compared to: `assert(a > b)`

Reply:
Why assertToBeGreaterThan and assertToBeLessThan  instead of assert(a > b)?
I can think of two reasons:
- It's more declarative(means saying what you want it to do, not how you want it done.)
- It allows the library to automatically provide more detailed assertion failed messages.